<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生选课系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/prefix1/u=2480412190,1647306576&amp;fm=26&amp;gp=0.jpg</normaloff>:/new/prefix1/u=2480412190,1647306576&amp;fm=26&amp;gp=0.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;学生Id ：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;学生 姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddStu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>增加学生信息</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStuId">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStuName">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddLes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>增加课程信息</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLessonId">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLessonName">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;课程Id ：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;课程名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddselect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>增加选课信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;学生Id ：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;课程Id ：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>240</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;分数 ：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbScore">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_lesid">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>240</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_stuid">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelet">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询 、统计</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnAddStu</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>addStudentInfo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddLes</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>addLessonInfo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddselect</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>addSelectionInfo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelet</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>deletWindow(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>findWindow(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>modifyWindow(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addStudentInfo(bool)</slot>
  <slot>addLessonInfo(bool)</slot>
  <slot>addSelectionInfo(bool)</slot>
  <slot>deletWindow(bool)</slot>
  <slot>findWindow(bool)</slot>
  <slot>modifyWindow(bool)</slot>
 </slots>
</ui>
